lib/compilation.sh

```diff
@@ -567,7 +567,7 @@ CUSTOM_KERNEL_CONFIG
 	git --no-pager -C ${kerneldir} log --abbrev-commit --oneline --no-patch --no-merges --date-order --date=format:'%Y-%m-%d %H:%M:%S' --pretty=format:'%C(black bold)%ad%Creset%C(auto) | %s | <%an> | <a href='$URL'%H>%H</a>' ${OLDHASHTARGET}..${hash} > "${HASHTARGET}.gitlog"
 
 	echo "${hash}" > "${HASHTARGET}.githash"
-	hash_watch_1=$(LC_COLLATE=C find -L "${SRC}/patch/kernel/${KERNELPATCHDIR}"/ -name '*.patch' -mindepth 1 -maxdepth 1 -printf '%s %P\n' 2> /dev/null | LC_COLLATE=C sort -n)
+	hash_watch_1=$(LC_COLLATE=C find -L "${SRC}/patch/kernel/${KERNELPATCHDIR}"/ -type f \( -iname \*.patch -o -iname \series.* \) -mindepth 1 -maxdepth 1 -printf '%s %P\n' 2> /dev/null | LC_COLLATE=C sort -n)
```

Comment:
If we want to calculate the hash of the applied patches, then we need to do it differently.
hash_watch_1=$hash_watch_1 "$(awk '$0 !~ /^#|^-|^$/{print $1}' "${SRC}/patch/kernel/${KERNELPATCHDIR}"/series.conf )"

Reply:
I have move this out for now.

Reply:
Here you use a git hash. And this is the right step.
https://github.com/armbian/build/blob/ef87a641c61e267a35e313a1a3ae51291b6f62f2/lib/compilation.sh#L546-L552

For patches, we need to use a git hash as well. It already exists and is always unambiguous. But you can not specify symbolic links, but only the real folder.
```
armbian-build> echo ${KERNELPATCHDIR}
sunxi-edge
armbian-build> echo $(git log --format="%h == %an" -1 -- patch/kernel/${KERNELPATCHDIR})
1e37959e5 == Igor Pečovnik
```
The last time the link was updated was by you
```
armbian-build> echo $(git log --format="%h == %an" -1 -- patch/kernel/archive/sunxi-5.15)
6a5363370 == The-going
```
I last updated the patches.